api/blueprints/drbCollection.py

```diff
@@ -89,6 +89,50 @@ def collectionCreate(user=None):
 
     return APIUtils.formatOPDS2Object(201, opdsFeed)
 
+@collection.route('/update/<uuid>', methods=['POST'])
+@validateToken
+def collectionUpdate(uuid, user=None):
+    logger.info('Updating current collection')
+
+    collectionData = request.json
+    dataKeys = collectionData.keys()
+
+    if len(set(dataKeys) & set(['title', 'creator', 'description'])) < 3\
```

Comment:
Maybe just a stylistic preference, but you could also use `issubset`: 
```
if {'title', 'creator', 'description'}.issubset(dataKeys):
```

I *think* that does the same thing, but up to you!

Reply:
That is a cleaner way of saying the same line. I'll implement it with this small change to dataKeys. 

`if {'title', 'creator', 'description'}.issubset(set(dataKeys))`